Miscellanous Puns (#301)
wesbos/dad-jokes: readme.md

---

**Q:** What was Elsa's advice about deprecated font commands in LaTeX

> (section 2.1.1 at [l2tabuen](https://ctan.math.utah.edu/ctan/tex-archive/info/l2tabu/english/l2tabuen.pdf))?

Author: ```suggestion
(section 2.1.1 at [l2tabuen](https://mirrors.ctan.org/info/l2tabu/english/l2tabuen.pdf))?
```

Distribute load by not pointing at one mirror

Author: Also for those opposing this: (La)TeX is a programming language. It is TURING COMPLETE.

> **A:** Let `\it` Go.

Author: Is this good? Is LaTeX on-topic? Should I let it go?

Reviewer: The fact that its linking to docs means its probably not well known and therefore not funny.

Also its a pdf so its not even linking to the actual section.

Author: LGTM.

> **Q:** What was Elsa's advice about deprecated font commands in LaTeX
> (section 2.1.1 at [l2tabuen](https://mirrors.ctan.org/info/l2tabu/english/l2tabuen.pdf))?
> 
> **A:** Let `\it` Go.
> 
> ---
> 

Author: ```suggestion
```

**A:** Dependency Day

---

> **Q:** What are iOS developers's favorite singer?

Reviewer: ```suggestion
**Q:** Who is the top musician among iOS developers?
```

What => Who

**A:** Taylor *Swift*.

---

The TypeScript haters say that `any` other language is better. I guess they have a bunch of `unknown` type safety bugs in their JS code.

---

A new JS framework just dropped, but I'd need time to properly React to it.

---

> **Q:** What do you call a modern JavaScript parrot running on an old browser?
> 
> **A:** A *poly*fill.
> 
> ---

Author: ```suggestion
```

> I like to `tkinter` with with various GUI frameworks.

Reviewer: What is this supposed to be? tinker?

Author: tkinter is a Python GUI module that is extremely dated. Tinker is what it's supposed to be punned for.

---

> Blindly trusting that 100% code coverage implies perfect tests is [`tox`](tox.wiki)ic.

Reviewer: ```suggestion
Blindly trusting that 100% code coverage implies no bugs is [`tox`](tox.wiki)ic.
```

**A:** Dependency Day

---

**Q:** What are iOS developers's favorite singer?

**A:** Taylor *Swift*.

---

The TypeScript haters say that `any` other language is better. I guess they have a bunch of `unknown` type safety bugs in their JS code.

---

A new JS framework just dropped, but I'd need time to properly React to it.

---

**Q:** What do you call a modern JavaScript parrot running on an old browser?

**A:** A *poly*fill.

---

> I like to `tkinter` with with various GUI frameworks.

Author: Two testing puns.

```suggestion
I like to `tkinter` with with various GUI frameworks.

---

Blindly trusting that 100% code coverage implies perfect tests is [`tox`](tox.wiki)ic.

---

**Q:** What do you call a programmer who writes a lot of tests?

**A:** An `assert`ive programmer.
```

> ---
> 

Author: ```suggestion
```

---

**Q:** Why do Jack and Jill like `git` so much?

> **A:** They use it to Go up the hill and `fetch` `<a>` pile of water `</a>`.

Author: Hmm... I'll see what I can do about http fetch. Maybe I'll mention some webscraper written in Golang.

> **A:** They use it to Go up the hill and `fetch` `<a>` pile of water `</a>`.

Author: Deleting this. as it should be split in a seperate PR to be added later

> **Q:** Why do Jack and Jill like `git` so much?
> 
> **A:** They use it to Go up the hill and `fetch` `<a>` pile of water `</a>`.

Author: ```suggestion
```

---

> **Q:** What are iOS developers' favorite singer?

Reviewer: This is good, surprised we don’t have it already!

Might be more puns to expand this one depending on her song titles.

Author: I have no idea about Taylor Swift, I just hear my middle school friends talking about it (refs profile).  So I'd need time to research about that.

Author: ```suggestion
**Q:** What are iOS developers's favorite singer?
```
Follows Chicago manual of style.

**A:** Taylor *Swift*.

---

> The TypeScript haters say that `any` other language is better. I guess they haven't went into the `unknown` to learn Brainfuck.

Reviewer: Is unknown a feature of BF or just trying to keep the TS pun going?

Author: ```suggestion
The TypeScript haters say that `any` other language is better. I guess they have a bunch of `unknown` type safety bugs in their JS code.
```
mentions js vs ts, and hard-to-debug of weakly typed languages, while keeping unknown pun

---

> A new feature was added to a JS framework, but I'd need time to properly React to it.

Reviewer: A new JS framework just dropped, but…

Author: ```suggestion
A new JS framework just dropped, but I'd need time to properly React to it.
```

---

**Q:** What do you call a modern JavaScript parrot running on an old browser?

**A:** A *poly*fill.

---

> I've been `tkinter`ing with various GUI frameworks recently.

Reviewer: I like to tkinter with various…
